FIAT/finite_element.py

```diff
@@ -137,10 +211,24 @@ def space_dimension(self):
         "Return the dimension of the finite element space."
         return self.poly_set.get_num_members()
 
-    def tabulate(self, order, points):
+    def tabulate(self, order, points, entity=None):
         """Return tabulated values of derivatives up to given order of
-        basis functions at given points."""
-        return self.poly_set.tabulate(points, order)
+        basis functions at given points.
+
+        Modified for trace element tabulation - passing information
```

Comment:
To elaborate a bit, this change to the docstring doesn't tell me what the "facet" argument is, or what happens if I pass `None`.  Perhaps you want something like:

```
def tabulate(self, order, points, entity=None):
    """Tabulate the finite element.

    :arg order: The maximum derivative to be tabulated.  All derivatives up to this value will be returned.
    :arg points: An iterable of points at which to tabulate the element.
    :kwarg entity: Optional entity indicating XXX, if ``None`` XXX
    """
    ...
```